doc/man/8/mount.ceph.rst

```diff
@@ -19,31 +18,35 @@ Description
 **mount.ceph** is a helper for mounting the Ceph file system on a Linux host.
 It serves to resolve monitor hostname(s) into IP addresses and read
 authentication keys from disk; the Linux kernel client component does most of
-the real work. In fact, it is possible to mount a non-authenticated Ceph file
-system without mount.ceph by specifying monitor address(es) by IP::
-
-        mount -t ceph 1.2.3.4:/ /mnt/mycephfs
-
-The first argument is the device part of the mount command. It includes host's
-socket and path within CephFS that will be mounted at the mount point. The
-socket, obviously, takes the form ip_address[:port]. If the port is not
-specified, the Ceph default of 6789 is assumed. Multiple monitor addresses can
-be passed by separating them by commas. Only one monitor is needed to mount
-successfully; the client will learn about all monitors from any responsive
-monitor. However, it is a good idea to specify more than one in case the one
-happens to be down at the time of mount.
-
-If the host portion of the device is left blank, then **mount.ceph** will
-attempt to determine monitor addresses using local configuration files
-and/or DNS SRV records. In similar way, if authentication is enabled on Ceph
-cluster (which is done using CephX) and options ``secret`` and ``secretfile``
-are not specified in the command, the mount helper will spawn a child process
-that will use the standard Ceph library routines to find a keyring and fetch
-the secret from it.
+the real work. To mount a Ceph file system use::
+
+  mount.ceph name@07fe3187-00d9-42a3-814b-72a4d5e7d5be.fs_name=/ /mnt/mycephfs -o mon_addr=1.2.3.4
+
+Mount helper can fill in the cluster FSID by reading the ceph configuration file.
+So, the mount command can be used as::
+
+  mount.ceph name@fs_name=/ /mnt/mycephfs -o mon_addr=1.2.3.4
```

Comment:
remove `mon_addr`?

Reply:
I see you mention that it can be not specified. Start with the conventional mount syntax we want to encourage: no `mon_addr` and no `fsid`.

Reply:
Agreed. With the new syntax, you shouldn't need to pass any options in most cases (which is even better).